tests/conftest.py

```diff
@@ -36,14 +36,14 @@ def pytest_collection_modifyitems(config, items):
 
 @pytest.fixture(scope="session", autouse=True)
 def start_doctest_collector(request):
-    matching = [s for s in set(sys.argv) if re.match(r'.*test_\w+\.py',s)]
-    if not matching: request.addfinalizer(stop_doctest_collector)
+#     matching = [s for s in set(sys.argv) if re.match(r'.*test_\w+\.py',s)]
```

Comment:
this should be removed

Reply:
Haha realized right after I opened the PR. Your review is too fast for me!